packages/product-react/src/Tabs/Tabs.scss

```diff
@@ -0,0 +1,66 @@
+$easing: cubic-bezier(0.694, 0.0482, 0.335, 1);
+$ray-color-black: black;
+$ray-color-blue-50: blue;
+$hover-border-size: 2px;
+$hover-animation-time: 0.33s;
```

Comment:
@adamraider Are you hard-pressed on seconds? What do you think about milliseconds?

Reply:
I don't feel strongly either way